app/src/androidTest/java/com/github/ashutoshgngwr/noice/fragment/WakeUpTimerFragmentTest.kt

```diff
@@ -312,15 +312,25 @@ class WakeUpTimerFragmentTest {
 
   @Test
   fun testSetSchedule_ShowNotifySnackBar() {
+    val context = ApplicationProvider.getApplicationContext<Context>()
     every { Preset.readAllFromUserPreferences(any()) } returns arrayOf(
       mockk(relaxed = true) {
-        every { id } returns "test-preset"
+        every { id } returns "test-preset-id"
         every { name } returns "test-preset"
       }
     )
+
+    every { WakeUpTimerManager.get(any()) } returns mockk {
+      every { presetID } returns "test-preset-id"
+      every { atMillis } returns System.currentTimeMillis() + 10000L
+      every { shouldUpdateMediaVolume } returns true
+      every { mediaVolume } returns 10
+    }
+
```

Comment:
Sorry, it appears that I made the wrong suggestion. I forgot to consider that the snack bar is only shown after the "Schedule" button is clicked. You can verify this behaviour in `testSetTimer()` test-case. Add your assertions after the action on line 208.

https://github.com/ashutoshgngwr/noice/blob/f1492f5fc1ff8788f5cdc5b5a1d0ab555b9a8630/app/src/androidTest/java/com/github/ashutoshgngwr/noice/fragment/WakeUpTimerFragmentTest.kt#L157-L235